vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -55,6 +58,138 @@ WinRTHttpResource::WinRTHttpResource(IHttpClient &&client) noexcept : m_client{s
 
 WinRTHttpResource::WinRTHttpResource() noexcept : WinRTHttpResource(winrt::Windows::Web::Http::HttpClient{}) {}
 
+#pragma region IWinRTHttpRequestFactory
+
+IAsyncOperation<HttpRequestMessage> WinRTHttpResource::CreateRequest(
+    HttpMethod &&method,
+    Uri &&uri,
+    winrt::Windows::Foundation::Collections::IMap<winrt::hstring, IInspectable> props) noexcept /*override*/ {
+  auto request = HttpRequestMessage{std::move(method), std::move(uri)};
+  for (auto prop : props) {
+    request.Properties().Insert(prop.Key(), prop.Value());
+  }
+
+  auto iReqArgs = request.Properties().Lookup(L"RequestArgs");
+  auto reqArgs = iReqArgs.as<RequestArgs>();
+  auto self = shared_from_this();
+
+  HttpMediaTypeHeaderValue contentType{nullptr};
+  string contentEncoding;
+  string contentLength;
+
+  // Headers are generally case-insensitive
+  // https://www.ietf.org/rfc/rfc2616.txt section 4.2
+  for (auto &header : reqArgs->Headers) {
+    if (boost::iequals(header.first.c_str(), "Content-Type")) {
```

Comment:
>iequals [](http://example.com/codeflow?start=15&length=7)

Assuming that this case insensitive compare is culture invariant, and not tied to current locale....

Reply:
Note: headers are asciii (i.e. 7-bits) so no chance of turkish i matching en-us i etc...